### PR TITLE
Add a `VideoView` for the local participant

### PR DESCRIPTION
**Pull request by marcisme**
This change is needed to display video for the local participant. It also updates formatting and wording to be more consistent. See in-line comments for more detail.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+xcuserdata/
```
Comment (marcisme) [inline .gitignore:1]: Ignoring `xcuserdata` prevents user-specific files with things like breakpoints and UI state from being committed.

File: DailyQuickstart.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -17,7 +17,7 @@
 /* End PBXBuildFile section */
 
 /* Begin PBXFileReference section */
-		221F387E29C64D7B008B8A6B /* Daily_iOS_Quickstart.app */ = {isa = PBXFileReference; explicitFileType = wrapper.application; includeInIndex = 0; path = Daily_iOS_Quickstart.app; sourceTree = BUILT_PRODUCTS_DIR; };
+		221F387E29C64D7B008B8A6B /* DailyQuickstart.app */ = {isa = PBXFileReference; explicitFileType = wrapper.application; includeInIndex = 0; path = DailyQuickstart.app; sourceTree = BUILT_PRODUCTS_DIR; };
```
Comment (marcisme) [inline DailyQuickstart.xcodeproj/project.pbxproj:20]: App names do not typically contain underscores, and we haven't included iOS in the name for other example code, so I thought we should be consistent here too.

File: DailyQuickstart.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 3]
```diff
@@ -6,7 +6,7 @@
       "location" : "https://github.com/daily-co/daily-client-ios",
       "state" : {
         "branch" : "main",
-        "revision" : "ef87257fb84876511c5a67338c18088377985dc8"
+        "revision" : "ecea4310731f68cec6a6573344aee38605ec6e33"
```
Comment (marcisme) [inline DailyQuickstart.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:9]: This updates to `0.8.0`.

File: DailyQuickstart/Info.plist [hunk 4]
```diff
@@ -25,9 +25,5 @@
 	<array>
 		<string>voip</string>
 	</array>
-    <key>NSCameraUsageDescription</key>
-           <string>Camera is necessary for transmitting video in a call</string>
-           <key>NSMicrophoneUsageDescription</key>
-        <string>Microphone is necessary for transmitting audio in a call</string>
```
Comment (marcisme) [inline DailyQuickstart/Info.plist:31]: These values were already set in the Build Settings, and Xcode removed them from here when I made another change.

File: DailyQuickstart/ViewController.swift [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+        super.viewDidLoad()
+
+        // Set the call client's delegate.
+        self.callClient.delegate = self
+
+        // Add the local participant's video view to the stack view.
+        self.participantsStack.addArrangedSubview(self.localVideoView)
+
+        // Disable the idle timer, so the screen will remain active while the app is in use.
+        UIApplication.shared.isIdleTimerDisabled = true
```
Comment (marcisme) [inline DailyQuickstart/ViewController.swift:38]: This is the line I mentioned in the other PR. A single line here and more detail about the notifications in the main doc seems like a good balance to me.

File: DailyQuickstart/ViewController.swift [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+    // The URL for the room to join.
+    private let roomURLString: String = "[YOUR_DAILY_ROOM_URL]"
+
+    // MARK: - Buttons
+
+    @IBOutlet private weak var cameraInputButton: UIButton!
+    @IBOutlet private weak var microphoneInputButton: UIButton!
+    @IBOutlet private weak var leaveRoomButton: UIButton!
+
+    // MARK: -
```
Comment (marcisme) [inline DailyQuickstart/ViewController.swift:23]: The `// MARK: -` syntax will create horizontal rules in the jump bar available via `ctrl-6` in Xcode, which helps group related properties visually.

File: DailyQuickstart/ViewController.swift [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+        // Determine whether the video input is from the camera or screen.
+        let cameraTrack = participant.media?.camera.track
+        let screenTrack = participant.media?.screenVideo.track
+        let videoTrack = screenTrack ?? cameraTrack
+        
+        // Set the track for this participant's video view.
+        videoView.track = videoTrack
+        
+        // Add this participant's video view to the dictionary.
+        self.videoViews[participant.id] = videoView
```
Comment (marcisme) [inline DailyQuickstart/ViewController.swift:117]: We don't get one of these calls for the local participant, so there's no way to add a view for them here.

**Review by lazeratops — APPROVED** [review]
LGTM, thank you for making these updates!

**Pull request merged**